### ISSUE TITLE
Add sigma, k1, k2, etc. as default parameters to tf.image.ssim_multiscale

### ISSUE THREAD

**Issue by veritas9872** [tensorflow/tensorflow#27138]
I previously requested a feature on tf.image.ssim for window size and sigma on this link. https://github.com/tensorflow/tensorflow/issues/26929 
Please read it for additional information.

I would like to request for variables of multiscale ssim (such as sigma, k1, k2, etc.) to be listed as defaults (instead of being fixed inside the script) as well.

**System information**
- TensorFlow version (you are using): 1.13
- Are you willing to contribute it (Yes/No): No. Sorry, I am not qualified to alter the code.

**Describe the feature and the current behavior/state.**

Currently, key parameters such as the sigma value are fixed when Tensorflow runs tf.image.ssim_multiscale. 

It would be great if all the variables mentioned in the original paper proposing MS-SSIM could be altered by the user.
(See Z. Wang, E. P. Simoncelli and A. C. Bovik, “Multi-scale structural similarity for image quality assessment,” IEEE Asilomar Conference Signals, Systems and Computers, Nov. 2003.)

This is especially important for those who wish to use MS-SSIM as a loss function as those variables have a significant influence on the character of the outputs.

**Will this change the current api? How?**
There will be almost no change. If the parameters which are currently fixed are set as default values, most people will never notice the difference.

**Who will benefit with this feature?**
Those who wish to use MS-SSIM as a loss function or wish to fine tune it on a deep learning model.

**Any Other info.**
I do not understand all of the features of MS-SSIM perfectly. I understand that a minimum window size of 176x176 is fixed and that K1 and K2 parameters are usually not altered. Perhaps these values should be set as keyword arguments (**kwargs) to hide unnecessary complexity.

**Comment by veritas9872** [comment]
On second thoughts, I don't think that this is necessary for MS-SSIM as much as it is for SSIM.
Closing issue.

**Comment by veritas9872** [comment]
Also, I found that everything was already going on without a separate issue.
